iml-action-client/src/lib.rs

```diff
@@ -2,35 +2,75 @@
 // Use of this source code is governed by a MIT-style
 // license that can be found in the LICENSE file.
 
+use bytes::buf::BufExt as _;
```

Comment:
Add iml-tracing to this crate and log in debug for interesting points (probably request / responses).

There are times when debugging code that knowing payloads / responses are useful.